packages/playwright-ct-svelte/package.json

```diff
@@ -30,10 +30,19 @@
   },
   "dependencies": {
     "@playwright/experimental-ct-core": "1.41.0-next",
-    "@sveltejs/vite-plugin-svelte": "^2.1.1"
+    "@sveltejs/vite-plugin-svelte": "^3.0.1"
   },
   "devDependencies": {
-    "svelte": "^3.55.1"
+    "svelte": "^4.2.8"
+  },
+  "peerDependencies": {
+    "svelte": "^4.0.0",
+    "@sveltejs/vite-plugin-svelte": "^3.0.0"
+  },
+  "peerDependenciesMeta": {
+    "@sveltejs/vite-plugin-svelte": {
```

Comment:
why is this dependency optional?

Reply:
It's already declared in `dependencies`, so the user doesn't need to install it - hence it's optional. But bad things will happen if the user installs v2 in their own project to use alongside this which is using v3, so that's why I made it a peerDependency as well. I'm not 100% sure how all the different package managers will handle this to be honest, but I'm relatively sure it shouldn't hurt at least. I'm hoping it would prevent people from using this alongside v2

Ultimately the best solution may be something like Storybook has with its support for various builders like Vite, webpack, etc. But in lieu of that major architectural change, this seemed like the best idea I could come up with